### ISSUE TITLE
Opendr renderer cannot initialize files

### ISSUE THREAD

**Issue by Geng1234** [polmorenoc/opendr#4]
runtime error

ValueError: operands could not be broadcast together with shapes (123984,3) (41328,3)

**Comment by sg-first** [comment]
Yes, there is an error calling the initGL function of the renderer:

ValueError: operands could not be broadcast together with shapes (123984,3) (41328,3)

We encountered this problem when we were trying to use this opendr as the back end of [up](https://github.com/classner/up)
